### ISSUE TITLE
Matplotlib - scaling of the y-axis

### ISSUE THREAD

**Issue by tf79** [Mri-monitoring/Mri-python-client#3]
- y-axis scales automatically but, because the loss is very high at the beginning you can't really figure out to which value the accuracy is converging. A two y-axis plot should fix this. 
- I noticed that in this config placing the legend in the lower left corner occludes  the early iteration phase. If you change to a two axis plot, put some offset to scale the accuracy axis, that when the legend is placed in the upper right corner nothing gets occluded.

**Comment by nharada1** [comment]
This one is a little tricky. The issue is that it's possible to have more than just two things showing in the plots (i.e. you could show learning rate, train error, test loss, etc), and so getting axes for each one could be hard. 

I'll have to think about this one. Options are:
- Just go from 0 to 1 like before
- Log scaling
- Dynamically re-scale all lines so that they go from [0 1]. This would make it really difficult to see the values though
- Use two scales, one for [0 1] and one for everything else. Accept an argument stating which variables are percentage scale and which are real value. This is probably the best option, but may require some architecture weirdness.

**Comment by tf79** [comment]
Last options sounds good. 
Might it be easier if the user would specify the output they want to see, as well as the scale? 
Not specified outputs could be ignored.

**Comment by tf79** [comment]
Could the location of the legend be moved out of the plot? 

This regarding to plotting so I hope it's fine to put it here: 
showing the max/peak and min values could help to compare plots with each other. 

I have a thousands of plots with different scaling and resolution of the image plots. Though it's very hard to compare them when it comes down to +-3 % or so.

**Comment by nharada1** [comment]
I've moved the legend to the bottom of the plot as well as added min and maxes in all the plots.
